examples/webgl_loader_xml_vtk.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
```

Comment:
Oh ok, so if consistency is not a problem , then Ill just go ahead and add it. Ill do it in some time.